umh-core/integration/docker_test_helpers.go

```
@@ -463,6 +463,27 @@ func printContainerLogs() {
+
+	// If the dir exists, remove it
+	if _, err := os.Stat(tmpDir); err == nil {
+		os.RemoveAll(tmpDir)
+	}
+
+	// Create the dir
+	err = os.MkdirAll(tmpDir, 0o777)
+	if err != nil {
+		fmt.Printf("Failed to create tmp dir: %v\n", err)
+	} else {
+		out, err = runDockerCommand("cp", "-r", "/data/logs", tmpDir)
```

golangci-lint (1.64.8)

ineffectual assignment to out

(ineffassign)